homeassistant/components/philips_air_purifier/client.py

```diff
@@ -0,0 +1,337 @@
+"""client provides a ReliableClient for Philips Air Purifiers."""
```

Comment:
There is quite some code here that could be moved to the pypi library perhaps? Any reason why you added this to HA core code?

Reply:
You're right, I moved the whole client to the library.

I only discovered quite late that I'd have to publish the library on PyPI myself and didn't rethink this.